master/get-deps.sh

```diff
@@ -4,5 +4,5 @@ go install github.com/bufbuild/buf/cmd/buf@v0.42.1
 go install golang.org/x/tools/cmd/goimports@v0.1.5
 go install github.com/goreleaser/goreleaser@v1.14.1
 go install github.com/swaggo/swag/cmd/swag@v1.8.9
-go install github.com/vektra/mockery/v2@v2.20.0
+go install github.com/vektra/mockery/v2@v2.42.0
```

Comment:
mockery had a panic in CI

I never got the mockery error locally though, but upgrading seems to fix it